### PR TITLE
This branch is to write Unit Tests for the OOP School Library build

### PR DESCRIPTION
**Pull request by steveWDamesJr**
## Unit Testing

Hello Reviewer 👋 
:sunglasses: This is @steveWDamesJr 's  and @levy002 's 3rd group milestone for the OOP-School-Library build. :sunglasses:

- [x] -Add spec_helper.rb to require necessary files
- [x] -Add rental_spec.rb with unit tests for rentals
- [x] -Add student_spec.rb with unit tests for students
- [x] -Add teacher_spec.rb with unit tests for teacher
- [x] -Add capitalize_spec_decorator.rb with unit tests for capitalize_decorator_spec
- [x] -Add classroom_spec.rb with unit tests for classroom
- [x] -Add person_spec.rb with unit tests for person
- [x] -debug rubocob linter errors
   
    
Unit Tests

Thank you for taking the time to review our PR :pray:

### CODE REVIEW

File: spec/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$LOAD_PATH << '.'
+require 'spec_helper'
+
+describe Book do
+  it 'Create a new book' do
+    @new_book = Book.new('The Great Book', 'The Awesome Author')
+    expect(@new_book.title).to eq('The Great Book')
+    expect(@new_book.author).to eq('The Awesome Author')
+  end
```
Comment (Lameck1) [inline spec/book_spec.rb:9]: - [x] Well done in writing this test. However, this is not exhaustive and does not offer proof that the functionalities of the `Book` class are working as expected.

  We need to test all the functions in the `Book` class. Please into the `Book` class and identify which functions are covered with tests and implement tests for them.

File: spec/person_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$LOAD_PATH << '.'
+require 'spec_helper'
+
+describe Person do
+  it 'Create a new person' do
+    @new_person = Person.new(19, 'Levy')
+    expect(@new_person.age).to eq(19)
+    expect(@new_person.name).to eq('Levy')
+    expect(@new_person.parent_permission).to eq(true)
+  end
```
Comment (Lameck1) [inline spec/person_spec.rb]: - [x] Same issue as above. Besides the initialize method, the `Person` class has other four functions we can cover with tests. Please, let's address this.

File: spec/classroom_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+$LOAD_PATH << '.'
+require 'spec_helper'
+
+describe Classroom do
+  it 'Create a new classroom' do
+    @new_class = Classroom.new('Ruby Class')
+    expect(@new_class.label).to eq('Ruby Class')
+  end
```
Comment (Lameck1) [inline spec/classroom_spec.rb:8]: - [x] Same issue as above. Here we can do just one more thing, assert that we can add a student to a classroom and that will be all.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @steveWDamesJr, & @levy002, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @steveWDamesJr and @levy002     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**